Fixes #29367 | update `ServiceWorkerRegistration.scope` and `ServiceWorkerRegistration.updateViaCache` to better reflect its relationship with `ServiceWorkerContainer.register()` (#30447)
mdn/content: files/en-us/web/api/serviceworkerregistration/index.md

- {{domxref("ServiceWorkerRegistration.pushManager")}} {{ReadOnlyInline}}
  - : Returns a reference to the {{domxref("PushManager")}} interface for managing push subscriptions including subscribing, getting an active subscription, and accessing push permission status.
- {{domxref("ServiceWorkerRegistration.scope")}} {{ReadOnlyInline}}

>   - :  A string representing a URL that defines a service worker's registration scope; that is, the range of URLs the service worker can control.

Author: ```suggestion
  - : Returns a string representing a URL that defines a service worker's registration scope; that is, the range of URLs the service worker can control.
```

- {{domxref("ServiceWorkerRegistration.sync")}} {{ReadOnlyInline}}
  - : Returns a reference to the {{domxref("SyncManager")}} interface, which manages background synchronization processes.
- {{domxref("ServiceWorkerRegistration.updateViaCache")}} {{ReadOnlyInline}}

>   - : Returns the value of the setting used to determine the circumstances in which the browser will consult the HTTP cache when it tries to update the service worker or any scripts that are imported via {{domxref("WorkerGlobalScope.importScripts", "importScripts()")}}.

Author: ```suggestion
  - : Returns the value of the setting used to determine the circumstances in which the browser will consult the HTTP cache when it tries to update the service worker or any scripts that are imported via {{domxref("WorkerGlobalScope.importScripts", "importScripts()")}}. It can be one of the following: `imports`, `all`, or `none`.
```

- {{domxref("ServiceWorkerRegistration.pushManager")}} {{ReadOnlyInline}}
  - : Returns a reference to the {{domxref("PushManager")}} interface for managing push subscriptions including subscribing, getting an active subscription, and accessing push permission status.
- {{domxref("ServiceWorkerRegistration.scope")}} {{ReadOnlyInline}}

>   - : Returns a string indicating the service worker's registration scope associated with the current service worker registration.

Reviewer: ```suggestion
  - :  A string representing a URL that defines a service worker's registration scope; that is, the range of URLs the service worker can control.
```

- {{domxref("ServiceWorkerRegistration.sync")}} {{ReadOnlyInline}}
  - : Returns a reference to the {{domxref("SyncManager")}} interface, which manages background synchronization processes.
- {{domxref("ServiceWorkerRegistration.updateViaCache")}} {{ReadOnlyInline}}
  - : Returns a string indicating the update via cache mode associated with the current service worker registration. It can be one of the following: `imports`, `all`, or `none`.
- {{domxref("ServiceWorkerRegistration.waiting")}} {{ReadOnlyInline}}

>   - : Returns a service worker whose state is `installed`. This is initially set to `null`

Reviewer: ```suggestion
  - : Returns a service worker whose state is `installed`. This is initially set to `null`.
```
